specification/botservice/resource-manager/Microsoft.BotService/preview/2023-09-15-preview/botservice.json

```diff
@@ -2,1786 +2,2181 @@
         },
         "zones": {
           "type": "array",
+          "description": "Entity zones",
           "items": {
             "type": "string"
           },
-          "description": "Entity zones",
           "readOnly": true
         }
-      }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
     },
-    "Sku": {
+    "BotChannel": {
+      "type": "object",
+      "description": "Bot channel resource definition",
       "properties": {
-        "name": {
-          "$ref": "#/definitions/SkuName",
-          "description": "The sku name"
+        "properties": {
+          "$ref": "#/definitions/Channel",
+          "description": "The set of properties specific to bot channel resource"
         },
-        "tier": {
-          "readOnly": true,
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets the SKU of the resource."
+        },
+        "kind": {
+          "$ref": "#/definitions/Kind",
+          "description": "Required. Gets or sets the Kind of the resource."
+        },
+        "etag": {
           "type": "string",
-          "description": "Gets the sku tier. This is based on the SKU name.",
-          "enum": [
-            "Free",
-            "Standard"
-          ],
-          "x-ms-enum": {
-            "name": "SkuTier",
-            "modelAsString": true
-          }
+          "description": "Entity Tag."
+        },
+        "zones": {
+          "type": "array",
+          "description": "Entity zones",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
         }
       },
-      "required": [
-        "name"
-      ],
-      "description": "The SKU of the cognitive services account."
-    },
-    "SkuName": {
-      "type": "string",
-      "description": "The name of SKU.",
-      "enum": [
-        "F0",
-        "S1"
-      ],
-      "x-ms-enum": {
-        "name": "SkuName",
-        "modelAsString": true
-      }
-    },
-    "Kind": {
-      "type": "string",
-      "description": "Indicates the type of bot service",
-      "enum": [
-        "sdk",
-        "designer",
-        "bot",
-        "function",
-        "azurebot"
-      ],
-      "x-ms-enum": {
-        "name": "Kind",
-        "modelAsString": true
-      },
-      "x-ms-mutability": [
-        "create",
-        "read"
-      ]
-    },
-    "Bot": {
-      "type": "object",
-      "description": "Bot resource definition",
       "allOf": [
         {
-          "$ref": "#/definitions/Resource"
-        }
-      ],
-      "properties": {
-        "properties": {
-          "$ref": "#/definitions/BotProperties",
-          "description": "The set of properties specific to bot resource"
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
```

Comment:
`location` should be optional for `Bot` `BotChannel` and `listChannelWithKeysResponse`